modules/viewing-security-information-in-red-hat-developer-hub.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * configuration/configuring-and-integrating-the-rhacs-plugin-with-red-hat-developer-hub.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-security-information-in-red-hat-developer-hub_{context}"]
+= Viewing security information in Red Hat Developer Hub
+
+By configuring and integrating the {rh-rhacs-first} plugin with {rh-rhdh-first}, you can access vulnerability data, assess risks, and take proactive security actions without leaving the {rh-rhdh} environment.
+
+.Prerequisites
+
+* You have enabled the {product-title-short} plugin installation in {rh-rhdh}.
++
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{rhdh-latest-version}/html/installing_and_viewing_plugins_in_red_hat_developer_hub/rhdh-installing-rhdh-plugins_title-plugins-rhdh-about#con-install-dynamic-plugin-helm_rhdh-installing-rhdh-plugins[Installing dynamic plugins using the Helm chart] ({rh-rhdh} documentation).
+
+.Procedure
+
+. Create an `app-config.yaml` file that contains the `proxy` and `acs` stanzas by using the following content:
++
+[source,yaml]
+----
+# ...
+proxy:
+  endpoints:
+    /acs:
+      target: ${ACS_API_URL}
+      headers:
+        authorization: Bearer ${ACS_API_KEY}
+acs:
+  acsUrl: ${ACS_API_URL}
+# ...
+----
+
+. To enable the {product-title-short} plugin, perform the following steps:
+.. Navigate to the dynamic plugins configuration file in your {rh-rhdh} setup, for example, `app-config.yaml`.
+.. To include the {product-title-short} plugin, add the following content to the configuration file, for example:
++
+[source,yaml]
+----
+# ...
+- package: https://github.com/RedHatInsights/backstage-plugin-advanced-cluster-security/releases/download/v0.1.1/redhatinsights-backstage-plugin-acs-dynamic-0.1.1.tgz
+  integrity: sha256-9JeRK2jN/Jgenf9kHwuvTvwTuVpqrRYsTGL6cpYAzn4=
+  disabled: false
+  pluginConfig:
+    dynamicPlugins:
+      frontend:
+        redhatinsights.backstage-plugin-acs:
+          entityTabs:
+            - path: /acs
+              title: RHACS
+              mountPoint: entity.page.acs
+          mountPoints:
+            - mountPoint: entity.page.acs/cards
+              importName: EntityACSContent
+              config:
+                layout:
+                  gridColumnEnd:
+                    lg: span 12
+                    md: span 12
+                    xs: span 12
+# ...
+----
+
+. To add annotations for entities in the {rh-rhdh} catalog, perform the following steps:
++
+[NOTE]
+====
+To display the vulnerability data, each entity in the {rh-rhdh} catalog must reference the {product-title-short} deployments.
```

Comment:
I would say the following for this:
`To display the vulnerability data, each component entity in the {rh-rhdh} catalog must reference the {product-title-short} deployments.`